src/contentProvider.ts

```diff
@@ -8,6 +8,12 @@ import { MavenProject } from "./explorer/model/MavenProject";
 import { getDependencyTree } from "./handlers/showDependenciesHandler";
 import { Utils } from "./utils/Utils";
 
+/**
+ * URI patterns.
+ * vscode-maven://dependencies/<pom-path>/Dependencies?<pom-path>
+ * vscode-maven://effective-pom/<pom-path>/EffectivePOM.xml?<pom-path>
+ * vscode-maven:///<pom-path-in-local-maven-repository>
```

Comment:
how about using a specific `authority` like the previous ones?

Reply:
yes, it's an option.

Reply:
Let's call it "local-repository"? What if later we want to open a project pom.xml in RO mode?

Reply:
> What if later we want to open a project pom.xml in RO mode?

no such requirement for the moment. let's rename / use another authority if later we need  do that.